README.md

```diff
@@ -1,24 +1,30 @@
+# Go Astilectron Bundler
```

Comment:
I don't feel like adding this header. Can you revert as well as all extra `#` in titles below?

Reply:
Yep, definitely! Sorry for going overboard on the README 😊